lib/simplifyExpression/stepThrough.js

```diff
@@ -84,6 +84,8 @@ function step(node) {
     breakUpNumeratorSearch,
     // e.g. 3/x * 2x/5 => (3 * 2x) / (x * 5)
     multiplyFractionsSearch,
+    // e.g. (2 + x) / 4 => 2/4 + x/4
+    breakUpNumeratorSearch,
```

Comment:
There was a previous suggestion from Evy to swap `breakUpNumberatorSearch` and `multiplyFractionSearch` before we discovered some of the issues blocking this PR.  This just looks like an oversight from merging master in.  I'll remove it